library/ssl_tls.c

```diff
@@ -2154,6 +2155,7 @@ static int ssl_decompress_buf( mbedtls_ssl_context *ssl )
 {
     int ret;
     unsigned char *msg_post = ssl->in_msg;
+    ptrdiff_t bytes_written = ssl->in_msg - ssl->in_buf;
```

Comment:
This isn't really `bytes_written` is it? But actually the bytes processed so far. Would this variable be better called `header_bytes`?

Reply:
Oh, you're right in this case - I just copied the output version. I'll change it then.